### PR TITLE
Complete all tasks in feature 3 - Vorleak Yek 

### PR DESCRIPTION
**Pull request by vorleakyek**
https://github.com/vorleakyek/code-journal/assets/76607726/c8cbd2e8-5d9f-4dd6-87e8-b6eeacaa3b92

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ let data = {
   nextEntryId: 1
 };
 
+// localStorage.clear();
```
Comment (kurtyazdizadeh) [inline js/data.js]: just remove the commented out code

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
just remove some commented out code, otherwise it looks good! 👍

File: js/data.js (hunk 1 above)
Comment (vorleakyek) [reply js/data.js]: Okay, I just removed it. Thanks!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**